[DOCS]: allow multiple searchable_snapshot actions in the same policy (#74679)
elastic/elasticsearch: docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

the data stream.

IMPORTANT: If the `searchable_snapshot` action is used in the hot phase the

> subsequent phases cannot define any of the `shrink`, `forcemerge` or `freeze`
> actions.

Reviewer: Nit: We tend to use an oxford comma. I'd also use `include` instead of `define`.
```suggestion
subsequent phases cannot include the `shrink`, `forcemerge`, or `freeze`
actions.
```
